migrate.py

```diff
@@ -183,6 +126,16 @@ def get_plugin_collection(plugin_name, plugin_type, spec):
     raise LookupError('Could not find "%s" named "%s" in any collection in the spec' % (plugin_type, plugin_name))
 
 
+def get_plugins_from_collection(collection, plugin_type, spec):
+    assert collection in spec
+    return [plugin.rsplit('/')[-1][:-3] for plugin in spec[collection].get(plugin_type, [])]
+
+
+def get_plugin_fqcn(namespace, collection, plugin_name):
+    return '%s.%s.%s' % (namespace, collection, plugin_name)
```

Comment:
So maybe 
```suggestion
    return f'{namespace}.{collection}.{plugin_name}'
```
for readability?